src/ui/components/FieldControl/FieldControl.svelte

```diff
@@ -75,6 +75,10 @@
       value={isDate(value) ? value : null}
       on:change={({ detail: value }) => (cachedValue = value)}
       on:blur={() => {
+        if (!cachedValue) {
+          onChange(cachedValue);
```

Comment:
After clearing within a date picker, the value would be null, which means the field will still in the note frontmatter with an empty value.